app/Console/Command/UpdateMunicipalName.php

```diff
@@ -0,0 +1,155 @@
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class UpdateMunicipalName extends AbstractCommand
+{
+    private const QUERY = 'query';
+
+    private const NEW_MUNICIPAL_NAME = 'new-municipal-name';
+
+    private const DRY_RUN = 'dry-run';
+
+    private ResultsGeneratorInterface $searchResultsGenerator;
+
+    private DocumentRepository $documentRepository;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService,
+        DocumentRepository $documentRepository
+    ) {
+        parent::__construct($commandBus);
+        $this->searchResultsGenerator = new ResultsGenerator(
+            $searchService,
+            new Sorting('created', 'asc'),
+            100
+        );
+        $this->documentRepository = $documentRepository;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('place:update-municipal-name')
+            ->setDescription('Update the location of all events from the given SAPI3 query to the given new location')
+            ->addArgument(
+                self::QUERY,
+                null,
+                'SAPI3 query for which places to update.'
+            )
+            ->addArgument(
+                self::NEW_MUNICIPAL_NAME,
+                null,
+                'The new name of the municipality in Dutch.'
+            )
+            ->addOption(
+                self::DRY_RUN,
+                null,
+                InputOption::VALUE_NONE,
+                'Execute the script as a dry run.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        $query = $input->getArgument(self::QUERY);
+        $newMunicipalName = $input->getArgument(self::NEW_MUNICIPAL_NAME);
+
+        if ($query === null || $newMunicipalName === null) {
+            $output->writeln('<error>Missing argument, the correct syntax is: place:update-municipal-name "sapi3_query" "new_municipal_name"</error>');
+            return self::FAILURE;
+        }
+
+        $query = str_replace('q=', '', $query);
+
+        $count = $this->searchResultsGenerator->count($query);
+
+        if ($count <= 0) {
+            $output->writeln('<error>No places found</error>');
+            return self::SUCCESS;
+        }
+
+
+        if (!$this->askConfirmation($input, $output, $count)) {
+            return self::SUCCESS;
+        }
+
+        foreach ($this->searchResultsGenerator->search($query) as $place) {
+            try {
+                $placeId = $place->getId();
+
+                $document = $this->documentRepository->fetch($placeId);
+                $jsonLd = Json::decodeAssociatively($document->getRawBody());
+
+                if (!isset(
+                    $jsonLd['address']['nl']['streetAddress'],
```

Comment:
So this means we will only update places with a Dutch address?

Reply:
Correct, we do not have (all the) translations for the all the (merged) municipalities.
This is primarly needed for UiTinVlaanderen(Dutch only). https://github.com/cultuurnet/udb3-frontend/blob/main/public/assets/citiesBE.json